docker/local-kms-seed/jwks.json

```diff
@@ -16,10 +9,10 @@
     },
     {
       "kty": "RSA",
-      "n": "rETceL76VNTt7_Vgsi3lDwpwQg6TpXZGKWRXeUYuZbJ8igKd7Jc8fQKJg0CYDZNxzdfPHrAiZ6MOo9VNWIAWdsj_F1XOAM0FowlIN_XpWIhp6d7Bj50DxVaSSO6iyNP4BzLe5EYPaBqH5jvI3dy-0tjXhtwCKcf0gaMuD0KX0YtnjuR5fkmfJy3Da7b4ALUleIIqMYLgnvAL_xUyJZoSdBpJW1-GYKSzQg3mjvc2wZYHCFEZz8CJNoZ2wjxjGNxalHFshBbPyw8KZ9-ZqG7jvSJBYkC1l-grZb_ZEpkLlfCGF_4l4zyGKPwn29mOed3C5sR09JUWTxvaGyZcou-KOQ",
+      "n": "rgYWtcWT6u1HnPDugjSyV2OnqndlPJ6HTzK0sCaMtx9i0zkYWzjD0328-WrECJ7yvglrDxsnOOD9-WF7jU4mI-CQch0922MsSeEKECFS2Ssl4jqrGMgoe1kNWlNBTqTPjBUwpNjoHF4xozh7JZOqxhsweidufn36VQH91PXH-N-YfEX0T1jjQZfV0wzmJteuqtdf1eqTQ7A6mXUDs-Hsgg-LrB7VJhB6Q6o_x-1O_WlOSTu_eajRoFVLiznHwFiUZFgpKv_KSfhAaIsfw3xgm_wDW9b-IqZYZQ2ZE7N_bi5slxPK93KjzUJNwHsFsVqQj3R7J1iJgtGk2g3Iwimpuw",
       "e": "AQAB",
       "alg": "RS256",
-      "kid": "3281b445-1700-4a46-a6ee-e7e02c9079ae"
+      "kid": "fcd1bab9-ae4d-49ce-9252-262db866e327"
     }
```

Comment:
Updating the local well known web keys since they changed after the migration of the dev env. Now this file contains only the key from the dev well known (https://dev.interop.pagopa.it/.well-known/jwks.json) and the key in the local KMS